content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -126,30 +129,20 @@ const Transcription = styled(Space).attrs({
   border-left: 20px solid ${props => props.theme.color('accent.lightBlue')};
 `;
 
-type Stop = {
-  standaloneTitle: string;
-  number?: number;
-  title: string;
-  image?: ImageType;
-  tombstone: prismicT.RichTextField;
-  caption: prismicT.RichTextField;
-  context?: prismicT.RichTextField;
-  transcription?: prismicT.RichTextField;
```

Comment:
This type is a subset of ExhibitionGuideComponent, so rather than duplicate it and try to keep the types in sync, I just replaced all uses with ExhibitionGuideComponent.